### PR TITLE
DO NOT MERGE

### PR DESCRIPTION
**Pull request by johnson-rl**
This is only a test

### CODE REVIEW

File: controllers/usersController.js [hunk 1]
```diff
@@ -5,12 +5,71 @@ const User       = require('../models/Users')
...
+
+
+//CREATE
+router.post('/', async (req, res)=>{
+    try{
+        const createdUser = await User.create(req.body)
+        console.log("////////////USER/////////////////")
+        console.log(createdUser)
+        console.log("////////////USER/////////////////")
+        res.redirect('/users')
```
Comment (johnson-rl) [inline controllers/usersController.js]: Change all of this please

File: controllers/usersController.js [hunk 2]
```diff
@@ -5,12 +5,71 @@ const User       = require('../models/Users')
...
 
+//NEW
+
+router.get('/new', (req, res) => {
+    res.render('users/new.ejs');
+})
+
+
+//CREATE
+router.post('/', async (req, res)=>{
```
Comment (johnson-rl) [inline controllers/usersController.js:33]: Should this be post?

File: models/Users.js [hunk 3]
```diff
@@ -2,7 +2,10 @@ const mongoose = require('mongoose');
 const Products = require('./Products')
 
 const userSchema = new mongoose.Schema({
-    username: {type: String, required: true, unique: true },
+    name: {type: String, required: false, unique: false },
+    email: {type: String, required: false, unique: false },
```
Comment (johnson-rl) [inline models/Users.js:6]: I think emails should be unique

**Review by johnson-rl — COMMENTED** [review]
left some notes for you

**Review by Negasie — COMMENTED** [review]
No!

**Pull request closed without merging**